pkg/interface/chat/src/js/components/lib/message.js

```diff
@@ -167,10 +167,10 @@ export class Message extends Component {
       let color = "#000000";
       if (contact) {
         name = (contact.nickname.length > 0)
-          ? contact.nickname : props.msg.author;
+          ? contact.nickname : "~" + props.msg.author;
         color = `#${uxToHex(contact.color)}`;
-      } 
-
+        console.log(name);
```

Comment:
remove this console.log

Reply:
I think he removed it already in 8f580aa3bb4e275e433bfe2dcd0c17d8e5f95faa